frontend/src/app/services/area-code.service.ts

```diff
@@ -0,0 +1,20 @@
+import { Injectable } from '@angular/core';
+import { HttpClient } from '@angular/common/http'; 
+import { Observable } from 'rxjs';
+@Injectable({
+  providedIn: 'root'
+})
+export class AreaCodeService {
+
+  constructor(private http: HttpClient) {
+    this.getAreaCodes().subscribe(data => {
+        console.log(data);
+    });
+}
+
+public getAreaCodes(): Observable<any> {
+    return this.http.get("./assets/data/AreaCodes.json");
```

Comment:
Instead of retrieving it as an asset why not just import it from a class?

Reply:
Noted. That will make data retrieving easier.